.github/workflows/cypress.yml

```
@@ -13,6 +13,9 @@ jobs:
         with:
           bundler-cache: true
 
+      - name: Run migrations
+        run: bin/rails db:migrate RAILS_ENV=development
```

```suggestion
        run: bin/rails db:migrate RAILS_ENV=test
```
This should be `test` or this is the bug we talked about?

---

This should be development